daemon/README.md

```diff
@@ -236,6 +236,7 @@ following environment variables will control its creation:
 * `WEIGHT` is the of the OSD when it is added to the CRUSH map (default is `1.0`)
 * `JOURNAL` is the location of the journal (default is the `journal` file inside the OSD data directory)
 * `HOSTNAME` is the name of the host; it is used as a flag when adding the OSD to the CRUSH map
+* `OSD_JOURNAL<osd-id>` optional - providing for each OSD directory to be used will prepare and mount the supplied block device
```

Comment:
I have this under the Ceph OSD directory section - refer to line 226. I put an example on line 260 under the "Multiple OSDs" subsection -- maybe it would help if I anchor-linked to it?